### DISCUSSION TITLE
Azuracast on Devuan Server?

### DISCUSSION THREAD

**Discussion by Tycho-X** [AzuraCast/AzuraCast#6559] [General]
Hello,

i would like to change vom Debian to Devuan because i am not a big Fan of the systemd system. Does anyone know of successfull Azuracast installations on Devuan 4/5? Any experiences?

One more question, can i use the AzuraCast docker image i am using on my Debian 11 server on Debian 12 too without modifications?

cheers....

**Comment by Piratenkanon** [comment]
Second question i can answer: here is running AzuraCast under Debian 12

**Comment by Tycho-X** [comment]
I did just install it on Devuan 5 (which is based on Debian 12). There had been one installer error "docker: 'compose' is not a docker command.". The installation will complete though and the docker image will start. I doubt i can safely ignore this error?

cheers...

**Reply by BusterNeece** [reply]
That usually means Docker Compose v2 didn't install correctly, but we support both Compose v1 and v2, so it should work fine.

**Reply by Tycho-X** [reply]
thx
